node/test/util/commit.js

```diff
@@ -59,131 +61,227 @@ const committer = co.wrap(function *(doAll, message, repos) {
     };
 });
 
-describe("commit", function () {
-    function makeCommitter(doAll, message) {
-        return function (repos, maps) {
-            return committer(doAll, message, repos, maps);
+describe("Commit", function () {
+    const FILESTATUS = RepoStatus.FILESTATUS;
+    describe("formatEditorPrompt", function () {
+        // Most of the work in this method is pass-through; just need to check
+        // that it's all wired up.
+
+        const cases = {
+            "basic form": {
+                status: new RepoStatus({
+                    currentBranchName: "master",
+                    staged: {
+                        "foo": FILESTATUS.ADDED,
+                    },
+                }),
+                match: /Changes to be committed/,
```

Comment:
I would actually like these to just describe the complete output.  match is error-prone in that it only checks for some deviations